README.md

```diff
@@ -45,6 +45,24 @@ to determine the level of pluralization.  Any additional arguments will be used
 
     (println (trsn "We found one cute puppy" "We found {0} cute puppies" 5))
 
+Use `format` to keep whitespace out of translations:
+
+    (str "Verification of client "
+      client
+      " provided by HTTP header "
+      header
+      "\nSee documentation for details\n")
+
+Replace this string with something like:
```

Comment:
The wording is a little weird. Maybe move `Replace this string` to before the example.